Scientist4JCore/src/test/java/com/github/rawls238/scientist4j/metrics/NoopMetricsProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.github.rawls238.scientist4j.metrics;
+
+import com.github.rawls238.scientist4j.metrics.MetricsProvider;
+
+/**
+ * A  minimal in-memory {@link MetricsProvider} implementation, suitable for test environments.
```

Comment:
I’m also 👌  to ditch this and use either Datadog or Micrometer in all of basic the tests.

Reply:
I think this is okay.